From 373aee3e7bdde9a701d643034c5a43b666755382 Mon Sep 17 00:00:00 2001
From: Rogerio Silva <rogerio.silva@nxp.com>
Date: Mon, 6 Apr 2020 14:29:14 -0300
Subject: [PATCH] weston.ini: Add NXP Demo Experience and terminal icons

Signed-off-by: Rogerio Silva <rogerio.silva@nxp.com>
---
 weston.ini.in | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/weston.ini.in b/weston.ini.in
index fec42917..f754dea6 100644
--- a/weston.ini.in
+++ b/weston.ini.in
@@ -19,3 +19,11 @@ idle-time=0
 
 [screen-share]
 command=@bindir@/weston --backend=rdp-backend.so --shell=fullscreen-shell.so --no-clients-resize
+
+[launcher]
+icon=/home/root/.nxp-demo-experience/icon/icon_demo_launcher.png
+path=HOME=/home/root /usr/bin/demoexperience --platform wayland-egl
+
+[launcher]
+icon=/usr/share/weston/icon_terminal.png
+path=/usr/bin/weston-terminal
-- 
2.17.1

