diff --git a/vit/i.MX8M_A53/ExApp/Makefile b/vit/i.MX8M_A53/ExApp/Makefile
index 9771346..06c6548 100644
--- a/vit/i.MX8M_A53/ExApp/Makefile
+++ b/vit/i.MX8M_A53/ExApp/Makefile
@@ -1,7 +1,4 @@
-CC          = aarch64-linux-gnu-gcc
-AR          = aarch64-linux-gnu-ar
-
-VERSION?=
+VERSION=4.7.5
 BLDDIR=build
 
 ifdef VERSION
@@ -36,6 +33,11 @@ CFLAGS=-std=gnu99 \
 	 -DARCH_AARCH64 \
 	 -march=armv8-a \
 	 -mcpu=cortex-a53 \
+	 -DRUN_ON_OTHER_THREAD \
+	 -pthread \
+	 -ldbus-1 \
+	 -ldbus-glib-1 \
+	 -lrt \
 	 $(IFLAGS)
 
 EXE_OBJ_FILES   = $(addprefix $(BLDDIR)/,$(patsubst %.c,%.o,$(notdir $(TEST_FILES))))
@@ -44,7 +46,7 @@ vpath %.c \
 	$(BLDDIR) \
 	ExApp
 
-EXE       = $(BLDDIR)/test_vit
+EXE       = $(BLDDIR)/btp_vit
 
 all : mkdir $(EXE)
 
